src/Compiler/TypedTree/TypedTreeOps.fs

```diff
@@ -3526,11 +3526,11 @@ let IsMatchingFSharpAttributeOpt g attrOpt (Attrib(tcref2, _, _, _, _, _, _)) =
 
 [<return: Struct>]
 let (|ExtractAttribNamedArg|_|) nm args = 
-    args |> List.tryPick (function AttribNamedArg(nm2, _, _, v) when nm = nm2 -> Some v | _ -> None) |> ValueOptionInternal.ofOption
-    
+    args |> List.tryPick (function AttribNamedArg(nm2, _, _, v) when nm = nm2 -> Some v | _ -> None) |> ValueOption.ofOption
+
```

Comment:
Off topic, but what is the point of this AP returning value option when a heap option is still created? Doesn't the conversion do nothing but add overhead?

Reply:
Yes.
A `tryPickV` equivalent would be better here.